src/components/pageheader/PageHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './pageheader.less';
+import { Typography } from 'antd';
+const { Title, Paragraph } = Typography;
+
+type PageHeaderProps = {
+  readonly pageTitle: string;
+  readonly pageSubtitle: string;
+  readonly subtitleColor: string;
+};
+
+const PageHeader: React.FC<PageHeaderProps> = ({
+  pageTitle,
+  pageSubtitle,
+  subtitleColor,
+}) => {
+  return (
+    <div className="header">
+      <Title id="title">{pageTitle}</Title>
+      <Paragraph className="subtitle" id={subtitleColor}>
```

Comment:
I think you should be able to use className here instead of id, perhaps something like 
```suggestion
      <Paragraph className={`subtitle ${subtitleColor}`}>
```